lib/marked-renderer.js

```diff
@@ -0,0 +1,13 @@
+// A custom renderer that avoids wrapping <img> tags in <p> tags
+const { marked } = require('marked')
+
+const markedRenderer = new marked.Renderer()
+
+markedRenderer.paragraph = function (text) {
+  if (text.startsWith('<img')) {
+    return `${text}\n`
+  }
+  return `<p>${text}</p>\n`
+}
+
+module.exports = markedRenderer
```

Comment:
We could consider sub-classing here instead, which better matches how things are done in Marked, and allows us to use `super` for cases other than images:

```suggestion
class DesignSystemRenderer extends marked.Renderer {
  paragraph (text) {
    if (text.startsWith('<img')) {
      return `${text}\n`
    }
    return super.paragraph(text)
  }
}

module.exports = new DesignSystemRenderer()
```

Reply:
Oooh, nice and neat!